sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1406,7 +1409,9 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
 
       static void ShareProtocolSettingsToXml(XmlWriter& writer, const ShareProtocolSettings& object)
       {
+        writer.Write(XmlNode{XmlNodeType::StartTag, "ProtocolSettings"});
```

Comment:
Add unit tests to capture regressions for such issues.

Reply:
This is already covered in `DISABLED_SetPremiumFileProperties`. However, the service is not ready to enable the test in the production environment.